tests/crypto_policy_test.go

```diff
@@ -21,10 +21,11 @@ import (
 	ssp "kubevirt.io/ssp-operator/api/v1beta3"
 	"kubevirt.io/ssp-operator/internal/common"
 	template_validator "kubevirt.io/ssp-operator/internal/operands/template-validator"
+	"kubevirt.io/ssp-operator/tests/decorators"
 	"kubevirt.io/ssp-operator/tests/env"
 )
 
-var _ = Describe("Crypto Policy", func() {
+var _ = Describe("Crypto Policy", decorators.Conformance, func() {
```

Comment:
I think this is not a good design. Someone can add a new test case to the `Describe` block and not notice that all the tests inside are labeled as `Conformance`.
Instead, it's better to label each test individually. A `DescribeTable` is also ok, because the test code is shared.

Reply:
You are right @akrejcir  generally best practice is to have decorators in Describe table, IT  but at the moment it has 4 tests and all tests are part of conformance so that's the reason I added in Describe.if you think i should move it to describe table i'll do it. Thanks for checking

Reply:
still applies?

Reply:
yes, I've missed this when approving. Thanks.